browser-test/src/northstar_end_to_end_enumerators.test.ts

```diff
@@ -0,0 +1,626 @@
+import {test, expect} from './support/civiform_fixtures'
+import {
+  AdminPrograms,
+  AdminQuestions,
+  enableFeatureFlag,
+  loginAsAdmin,
+  logout,
+  validateAccessibility,
+  validateScreenshot,
+  waitForPageJsLoad,
+} from './support'
+import {Page} from 'playwright'
+
+test.describe('End to end enumerator test', {tag: ['@northstar']}, () => {
+  const programName = 'Ete enumerator program'
+
```

Comment:
Question(Blocking): Should the admin tests execute with North Star enabled? If so, then:
* create a `beforeEach` block here that enables North Star UI.
* Delete `enableFeatureFlag` from line 73